scripts/certgen.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# -----
+# Purpose: This Script is used to Generate BIE kafa Compatible Base64 Encoded
+#   Certificate String. This Script also generates two passwords truststore and keystore passwords
+#
+# Output: Output of this script is output.json. 
+#   Output.json Contains 4 key value pairs which are then uploaded to vault
+#
+#   Following keys/values are generated
+#       BIE_KAFKA_KEYSTORE_INBASE64
+#       BIE_KAFKA_KEYSTORE_PASSWORD
+#       BIE_KAFKA_TRUSTSTORE_INBASE64
+#       BIE_KAFKA_TRUSTSTORE_PASSWORD
+# Pre-Req:
+#     - Download Certs locally. Link is provided in wiki https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/VRO-Secrets
+#       VA-Internal-S2-ICA4.cer 
+#       VA-Internal-S2-ICA19.cer
+#       VA-Internal-S2-ICA11.cer 
+#       VA-Internal-S2-RCA2.cer
+# Input: 
+#    $1 for Environment, Example dev/qa/prod
+#    $2 for BIE  provided Cert
+#    $3 for BIE  provided key
+#    $4 Kafka env prefix Example, dev,ivv,pr,pp
+# 
+#    Additional information is provided in https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/VRO-Secrets
+# - ----
+# Function to generate a random password
+generate_password() {
+    # Define characters for password generation
+    upper="ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    lower="abcdefghijklmnopqrstuvwxyz"
+    numbers="0123456789"
+    special="@#\$%^&*()_-+="
+
+    # Combine all characters into one set
+    all_chars="${upper}${lower}${numbers}${special}"
+
+    # Generate a random password of length 15 using openssl
+    password=$(openssl rand -base64 20 | tr -dc "$all_chars" | head -c 16)
+    echo "$password"
+}
+
+export KEYSTORE_PWD=$(generate_password)
+export TRUSTSTORE_PWD=$(generate_password)
+rm -rf bip.truststore.jks keystore.p12 bip.truststore.p12 passwd output.json
```

Comment:
It's safer to let user delete files. 
```suggestion
```

Reply:
Added with purpose. If we have to re run the same script with new password. then we get password mismatch error. its better to cleanup all temp script and start from begining